lib/tasks/git-init.js

```diff
@@ -24,14 +24,19 @@ class GitInitTask extends Task {
     return this._gitVersion().then(() => {
       gitVersionWorked = true;
 
-      return this._gitInit()
+      return this._gitEmailConfigured()
```

Comment:
I'd change the name to something like `getCommiterEmail` & `getComitterName` but that's my personal preference. @kellyselden @Turbo87 thoughts?